be/src/runtime/exec_env.cpp

```diff
@@ -713,6 +713,8 @@ void ExecEnv::destroy() {
     SAFE_DELETE(_lake_update_manager);
     SAFE_DELETE(_lake_replication_txn_manager);
     SAFE_DELETE(_cache_mgr);
+    _dictionary_cache_pool.reset();
```

Comment:
Is there a difference between manual destruction and automatic destruction?

Reply:
the order of destruction. With this, it is ensured that the pool will be destroyed before main exit, so that all other global static instances are still available. After exit main and enters global exit procedure, it is out of controller which global static instances are available over the others.